src/test/java/net/sf/jabref/logic/exporter/ModsExportFormatTestFiles.java

```diff
@@ -0,0 +1,82 @@
+package net.sf.jabref.logic.exporter;
+
+import java.io.File;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import net.sf.jabref.BibDatabaseContext;
+import net.sf.jabref.Globals;
+import net.sf.jabref.logic.importer.fileformat.BibtexImporter;
+import net.sf.jabref.model.entry.BibEntry;
+import net.sf.jabref.preferences.JabRefPreferences;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameter;
+import org.junit.runners.Parameterized.Parameters;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(Parameterized.class)
+public class ModsExportFormatTestFiles {
+
+    public Charset charset;
```

Comment:
Please not! When you test sth with export/import, please always use StandardCharsets.UTF-8.
Otherwise gradle build on win will complain (because the console charset is different from utf-8 on win)